server/test/com/cloud/network/firewall/FirewallManagerTest.java

```diff
@@ -52,9 +60,9 @@
 import com.cloud.network.rules.FirewallRuleVO;
 import com.cloud.utils.component.ComponentContext;
 
-@Ignore("Requires database to be set up")
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(locations = "classpath:/testContext.xml")
+//@Ignore("Requires database to be set up")
+@RunWith(MockitoJUnitRunner.class)
+//@ContextConfiguration(locations = "classpath:/testContext.xml")
```

Comment:
When you look at the file, it's full of commented lines and blocks.. So doesn't really help in removing only these two :-s